pkg/driver/aws-ebs/aws_ebs_tags_controller.go

```diff
@@ -0,0 +1,511 @@
+package aws_ebs
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog/v2"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorapi "github.com/openshift/api/operator/v1"
+	"github.com/openshift/csi-operator/pkg/clients"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	awsSecretNamespace     = "openshift-cluster-csi-drivers"
+	awsSecretName          = "ebs-cloud-credentials"
+	infrastructureResource = "cluster"
+	driverName             = "ebs.csi.aws.com"
+	tagHashAnnotationKey   = "ebs.csi.aws.com/volume-tags-hash"
```

Comment:
we should keep this in openshift namespace. So something like - `ebs.openshift.io/volume-tags-hash`